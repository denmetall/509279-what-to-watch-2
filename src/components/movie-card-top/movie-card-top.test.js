import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardTop from "./movie-card-top.jsx";
import filmsMock from '../../mocks/films';

it(`Components MovieCardTop renders correctly`, () => {
  const tree = renderer
    .create(<MovieCardTop film={filmsMock[0]}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
