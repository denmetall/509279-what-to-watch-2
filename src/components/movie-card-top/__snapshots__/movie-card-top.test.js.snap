// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components MovieCardTop renders correctly 1`] = `
<div
  className="movie-card__wrap movie-card__translate-top"
>
  <div
    className="movie-card__info"
  >
    <div
      className="movie-card__poster movie-card__poster--big"
    >
      <img
        alt="The Grand Budapest Hotel poster"
        height="327"
        src="img/the-grand-budapest-hotel-poster.jpg"
        width="218"
      />
    </div>
    <div
      className="movie-card__desc"
    >
      <nav
        className="movie-nav movie-card__nav"
      >
        <ul
          className="movie-nav__list"
        >
          <li
            className="movie-nav__item"
          >
            <a
              className="movie-nav__link"
              href="#"
            >
              Overview
            </a>
          </li>
          <li
            className="movie-nav__item movie-nav__item--active"
          >
            <a
              className="movie-nav__link"
              href="#"
            >
              Details
            </a>
          </li>
          <li
            className="movie-nav__item"
          >
            <a
              className="movie-nav__link"
              href="#"
            >
              Reviews
            </a>
          </li>
        </ul>
      </nav>
      <div
        className="movie-card__text movie-card__row"
      >
        <div
          className="movie-card__text-col"
        >
          <p
            className="movie-card__details-item"
          >
            <strong
              className="movie-card__details-name"
            >
              Director
            </strong>
            <span
              className="movie-card__details-value"
            >
              Wes Andreson
            </span>
          </p>
          <p
            className="movie-card__details-item"
          >
            <strong
              className="movie-card__details-name"
            >
              Starring
            </strong>
            <span
              className="movie-card__details-value"
            >
              Bill Murray, 
              <br />
              Edward Norton, 
              <br />
              Jude Law, 
              <br />
              Willem Dafoe, 
              <br />
              Saoirse Ronan, 
              <br />
              Tony Revoloru, 
              <br />
              Tilda Swinton, 
              <br />
              Tom Wilkinson, 
              <br />
              Owen Wilkinson, 
              <br />
              Adrien Brody, 
              <br />
              Ralph Fiennes, 
              <br />
              Jeff Goldblum
            </span>
          </p>
        </div>
        <div
          className="movie-card__text-col"
        >
          <p
            className="movie-card__details-item"
          >
            <strong
              className="movie-card__details-name"
            >
              Run Time
            </strong>
            <span
              className="movie-card__details-value"
            >
              1h 39m
            </span>
          </p>
          <p
            className="movie-card__details-item"
          >
            <strong
              className="movie-card__details-name"
            >
              Genre
            </strong>
            <span
              className="movie-card__details-value"
            >
              Comedy
            </span>
          </p>
          <p
            className="movie-card__details-item"
          >
            <strong
              className="movie-card__details-name"
            >
              Released
            </strong>
            <span
              className="movie-card__details-value"
            >
              2014
            </span>
          </p>
        </div>
      </div>
      <div
        className="movie-rating"
      >
        <div
          className="movie-rating__score"
        >
          8,9
        </div>
        <p
          className="movie-rating__meta"
        >
          <span
            className="movie-rating__level"
          >
            Very good
          </span>
          <span
            className="movie-rating__count"
          >
            240 ratings
          </span>
        </p>
      </div>
      <div
        className="movie-card__text"
      >
        <p>
          In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.
        </p>
        <p>
          Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.
        </p>
        <p
          className="movie-card__director"
        >
          <strong>
            Director: Wes Andreson
          </strong>
        </p>
        <p
          className="movie-card__starring"
        >
          <strong>
            Starring: Bill Murray, Edward Norton, Jude Law, Willem Dafoe and other
          </strong>
        </p>
      </div>
      <div
        className="movie-card__reviews movie-card__row"
      >
        <div
          className="movie-card__reviews-col"
        >
          <div
            className="review"
          >
            <blockquote
              className="review__quote"
            >
              <p
                className="review__text"
              >
                Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the directors funniest and most exquisitely designed movies in years.
              </p>
              <footer
                className="review__details"
              >
                <cite
                  className="review__author"
                >
                  Kate Muir
                </cite>
                <time
                  className="review__date"
                  dateTime="2016-12-24"
                >
                  December 24, 2016
                </time>
              </footer>
            </blockquote>
            <div
              className="review__rating"
            >
              8,9
            </div>
          </div>
          <div
            className="review"
          >
            <blockquote
              className="review__quote"
            >
              <p
                className="review__text"
              >
                Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the directors funniest and most exquisitely designed movies in years.
              </p>
              <footer
                className="review__details"
              >
                <cite
                  className="review__author"
                >
                  Kate Muir
                </cite>
                <time
                  className="review__date"
                  dateTime="2016-12-24"
                >
                  December 24, 2016
                </time>
              </footer>
            </blockquote>
            <div
              className="review__rating"
            >
              8,9
            </div>
          </div>
          <div
            className="review"
          >
            <blockquote
              className="review__quote"
            >
              <p
                className="review__text"
              >
                Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the directors funniest and most exquisitely designed movies in years.
              </p>
              <footer
                className="review__details"
              >
                <cite
                  className="review__author"
                >
                  Kate Muir
                </cite>
                <time
                  className="review__date"
                  dateTime="2016-12-24"
                >
                  December 24, 2016
                </time>
              </footer>
            </blockquote>
            <div
              className="review__rating"
            >
              8,9
            </div>
          </div>
        </div>
        <div
          className="movie-card__reviews-col"
        >
          <div
            className="review"
          >
            <blockquote
              className="review__quote"
            >
              <p
                className="review__text"
              >
                Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the directors funniest and most exquisitely designed movies in years.
              </p>
              <footer
                className="review__details"
              >
                <cite
                  className="review__author"
                >
                  Kate Muir
                </cite>
                <time
                  className="review__date"
                  dateTime="2016-12-24"
                >
                  December 24, 2016
                </time>
              </footer>
            </blockquote>
            <div
              className="review__rating"
            >
              8,9
            </div>
          </div>
          <div
            className="review"
          >
            <blockquote
              className="review__quote"
            >
              <p
                className="review__text"
              >
                Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the directors funniest and most exquisitely designed movies in years.
              </p>
              <footer
                className="review__details"
              >
                <cite
                  className="review__author"
                >
                  Kate Muir
                </cite>
                <time
                  className="review__date"
                  dateTime="2016-12-24"
                >
                  December 24, 2016
                </time>
              </footer>
            </blockquote>
            <div
              className="review__rating"
            >
              8,9
            </div>
          </div>
          <div
            className="review"
          >
            <blockquote
              className="review__quote"
            >
              <p
                className="review__text"
              >
                Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the directors funniest and most exquisitely designed movies in years.
              </p>
              <footer
                className="review__details"
              >
                <cite
                  className="review__author"
                >
                  Kate Muir
                </cite>
                <time
                  className="review__date"
                  dateTime="2016-12-24"
                >
                  December 24, 2016
                </time>
              </footer>
            </blockquote>
            <div
              className="review__rating"
            >
              8,9
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
